### DISCUSSION TITLE
BLE 5.2 5.4   Coded Phy (Long Range BLE)  Commands

### DISCUSSION THREAD

**Discussion by joeygold** [hbldh/bleak#1636] [Ideas]
Application:  BLE Long range
I can set my external BLE Module to change from 1M or 2M Phy to  Coded Phy for long range 1. During Advertising 2. During connection
I can interrogate my BLE Module to see which Phy is in use during connection 

Question
Does Bleak allow the changing or reporting of which Phy is in use on Windows/Mac/Linux?

**Comment by dlech** [comment]
No, this isn't possible right now.

Related issue: https://github.com/hbldh/bleak/issues/1225
